plugins/woocommerce-admin/client/marketing/overview-multichannel/LearnMarketing.tsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { Pagination, EmptyContent } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { CollapsibleCard, ReadBlogMessage } from '../components';
+import { STORE_KEY } from '../data/constants';
+import './LearnMarketing.scss';
+
+type Post = {
+	author_avatar: string;
+	author_name: string;
+	date: string;
+	image: string;
+	link: string;
+	title: string;
+};
+
+type PostTileProps = {
+	post: Post;
+};
+
+const PlaceholderPostTile = () => {
+	return (
+		<div className="woocommerce-marketing-learn-marketing-card__post">
+			<div className="woocommerce-marketing-learn-marketing-card__post-img woocommerce-marketing-learn-marketing-card__post-img--placeholder"></div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-title woocommerce-marketing-learn-marketing-card__post-title--placeholder"></div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-description woocommerce-marketing-learn-marketing-card__post-description--placeholder"></div>
+		</div>
+	);
+};
+
+const PostTile: React.FC< PostTileProps > = ( { post } ) => {
+	return (
+		<a
+			className="woocommerce-marketing-learn-marketing-card__post"
+			href={ post.link }
+			target="_blank"
+			rel="noopener noreferrer"
+			onClick={ () => {
+				recordEvent( 'marketing_knowledge_article', {
+					title: post.title,
+				} );
+			} }
+		>
+			<div className="woocommerce-marketing-learn-marketing-card__post-img">
+				{ post.image && <img src={ post.image } alt="" /> }
+			</div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-title">
+				{ post.title }
+			</div>
+			<div className="woocommerce-marketing-learn-marketing-card__post-description">
+				{
+					// translators: %s: author's name.
+					sprintf( __( 'By %s', 'woocommerce' ), post.author_name )
+				}
+				{ post.author_avatar && (
+					<img
+						src={ post.author_avatar.replace( 's=96', 's=32' ) }
+						alt=""
+					/>
+				) }
+			</div>
+		</a>
+	);
+};
+
+const blogPostCategory = 'marketing';
+const perPage = 2;
```

Comment:
💅  I would name these constants using UPPERCASE as STORE_KEY does

Reply:
Done in [`f611b0b` (#34264)](https://github.com/woocommerce/woocommerce/pull/34264/commits/f611b0bd4ef3b25513c6bd29314ac44f29f53680).